examples/usecases/multi-gpu/install_sparse_operation_kit.sh

```diff
@@ -11,6 +11,6 @@ rm -rf hugectr/
 git clone https://github.com/NVIDIA-Merlin/HugeCTR.git hugectr
 
 cd hugectr/sparse_operation_kit/
-python setup.py install
+python setup.py develop --no-deps
```

Comment:
if horovod is installed already in the container, I wonder why this is trying to install horovod again. does it require a higher version that what we have already?

Reply:
thanks for the explanation @edknv